src/main/java/com/hp/application/automation/tools/rest/RestClient.java

```diff
@@ -4,13 +4,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.net.Authenticator;
-import java.net.HttpURLConnection;
-import java.net.InetSocketAddress;
-import java.net.PasswordAuthentication;
-import java.net.Proxy;
-import java.net.URL;
-import java.net.URLConnection;
```

Comment:
Please avoid such changes - explicit import makes life easier to catch import issues